src/fides/api/ops/task/graph_task.py

```diff
@@ -114,6 +116,20 @@ def result(*args: Any, **kwargs: Any) -> Any:
                     )
                     self.log_skipped(action_type, exc)
                     return default_return
+                except (SkippingConsentPropagation,) as exc:
```

Comment:
nit: does the `,` mean something here i'm just not aware of? or was this just a copy/paste blip? :)

Reply:
thanks for highlighting, just copy paste from above where I split it out to handle this exception separately.  This format is for when you're catching more than once exception.